app/upgrades/v5/constants.go

```diff
@@ -17,6 +21,15 @@ const (
 	ContributorAddrFrom = "evmos13cf9npvns2vhh3097909mkhfxngmw6d6eppfm4"
 	// ContributorAddrTo is the new address of an early contributor
 	ContributorAddrTo = "evmos1hmntpkn623y3vl0nvzrazvq4rqzv3xa74l40gl"
+	// AvgBlockTime defines the new expected average blocktime on mainnet and testnet
+	//
+	// CONTRACT: in order for AvgBlockTime to represent an accurate value on-chain, validator nodes
```

Comment:
@fedekunze I've seen the `CONTRACT` notation a few times. Could you explain when to use it?

Reply:
I personally use it as a `MUST` like whenever something needs to be enforced, but not on the same function, constant, etc.

@alexanderbez can also give his opinion

Reply:
It typically means to state the invariants the function or method takes, either in its inputs and/or outputs. Re MUST vs CONTRACT, I honestly don't have a strong opinion, other than just being consistent.